### ISSUE TITLE
[RESEARCH] Investigate performing a code coverage analysis w.r.t. existing tests.

### ISSUE THREAD

**Issue by ndouglas** [department-of-veterans-affairs/va.gov-cms#9881]
## Description
In order to move forward with any sort of QA initiative, we need to first determine where we stand currently.  A good place to start is by determining which code currently has unit test coverage.  This is approximately 0%.  We can factor in the integration and behavioral tests, if we find a good way of doing so, but that might be challenging and extremely lengthy because of how slowly code will execute with that sort of profiling enabled.

This should be treated as a research spike; it might involve throwing a few different possible approaches at the wall and seeing which (if any) stick.  It's possible that none might work and that we don't get much useful information about our code coverage.  That is in itself significant; we can't make very many positive claims about the quality of our codebase if we can't even instrument it.

To follow up, I'll open issues to deal with the findings.  These may take the form of:
- issues to expand coverage in problematic areas
- issues to improve instrumentation for code coverage
- issues to enable instrumentation
- issues to improve the codebase so that instrumentation is possible

## Acceptance Criteria
- [x] A solution or solutions for analyzing test coverage has been investigated.
- [ ] Findings have been documented and followup issues have been created.

**Comment by ndouglas** [comment]
I'm giving up on recording code coverage outside of unit tests.  I thought this was a long shot, and I'm not seeing anything to change my mind.  I'm going to block this on #10053, since that will be the core mechanism for recording code coverage moving forward.

**Comment by ndouglas** [comment]
Just kidding, this blocks #10555 , not the other way around.

**Comment by ndouglas** [comment]
**TL;DR**:

![Screen Shot 2022-09-27 at 10 31 43 AM](https://user-images.githubusercontent.com/1318579/192557127-52c3bbc8-cd7a-47fd-951f-125ab90a5538.png)

As suspected, the amount of coverage is very, very small (this output doesn't individually list files, functions, etc that aren't tested at all).

**Comment by ndouglas** [comment]
For comparison, these are the obvious-at-first-glance files with custom PHP code:

```
$ find ./docroot/modules/custom -type f -name "*.php" -or -name "*.module" -or -name "*.inc"
./docroot/modules/custom/va_gov_user/va_gov_user.module
./docroot/modules/custom/va_gov_user/src/Form/VaGovUserMigrateSourceUiForm.php
./docroot/modules/custom/va_gov_user/src/Plugin/Block/UserSections.php
./docroot/modules/custom/va_gov_user/src/Routing/RouteSubscriber.php
./docroot/modules/custom/va_gov_user/src/Service/UserPermsService.php
./docroot/modules/custom/va_gov_user/src/EventSubscriber/UserImport.php
./docroot/modules/custom/va_gov_clp/va_gov_clp.module
./docroot/modules/custom/va_gov_resources_and_support/va_gov_resources_and_support.module
./docroot/modules/custom/va_gov_resources_and_support/src/Plugin/Block/WhereDoesThisAppear.php
./docroot/modules/custom/va_gov_preview/va_gov_preview.module
./docroot/modules/custom/va_gov_preview/src/Encoder/StaticEncoder.php
./docroot/modules/custom/va_gov_preview/src/StaticServiceProvider.php
./docroot/modules/custom/va_gov_lovell/va_gov_lovell.module
./docroot/modules/custom/va_gov_lovell/src/EventSubscriber/LovellEventSubscriber.php
./docroot/modules/custom/va_gov_api/src/Resources/BannerAlerts.php
./docroot/modules/custom/va_gov_api/src/Plugin/openapi/OpenApiGenerator/VAOpenApiGenerator.php
./docroot/modules/custom/va_gov_consumers/va_gov_consumers.module
./docroot/modules/custom/va_gov_consumers/src/Git/GithubFactory.php
./docroot/modules/custom/va_gov_consumers/src/Git/Git.php
./docroot/modules/custom/va_gov_consumers/src/Git/GitInterface.php
./docroot/modules/custom/va_gov_consumers/src/Git/GithubInterface.php
./docroot/modules/custom/va_gov_consumers/src/Git/GitFactory.php
./docroot/modules/custom/va_gov_consumers/src/Git/GithubAdapter.php
./docroot/modules/custom/va_gov_workflow/va_gov_workflow.module
./docroot/modules/custom/va_gov_workflow/src/Service/Flagger.php
./docroot/modules/custom/va_gov_workflow/src/Service/WorkflowContentControl.php
./docroot/modules/custom/va_gov_workflow/src/EventSubscriber/EntityEventSubscriber.php
./docroot/modules/custom/va_gov_banner/va_gov_banner.module
./docroot/modules/custom/va_gov_banner/src/Plugin/Validation/Constraint/RequireScopeValidator.php
./docroot/modules/custom/va_gov_banner/src/Plugin/Validation/Constraint/RequireScope.php
./docroot/modules/custom/va_gov_banner/src/EventSubscriber/EntityEventSubscriber.php
./docroot/modules/custom/va_gov_govdelivery/va_gov_govdelivery.module
./docroot/modules/custom/va_gov_govdelivery/src/Service/ProcessStatusBulletin.php
./docroot/modules/custom/va_gov_govdelivery/src/EventSubscriber/EntityEventSubscriber.php
./docroot/modules/custom/va_gov_graphql/va_gov_graphql.module
./docroot/modules/custom/va_gov_graphql/src/Access/MenuLinkContentAccessHandler.php
./docroot/modules/custom/va_gov_graphql/src/Plugin/GraphQL/Fields/SiteMenus.php
./docroot/modules/custom/va_gov_graphql/src/Plugin/GraphQL/Fields/ModerationState.php
./docroot/modules/custom/va_gov_graphql/src/Plugin/GraphQL/Fields/SideNavMenus.php
./docroot/modules/custom/va_gov_graphql/src/Controller/ExplorerController.php
./docroot/modules/custom/va_gov_graphql/src/Routing/ExplorerRoutes.php
./docroot/modules/custom/va_gov_graphql/src/Routing/RouteSubscriber.php
./docroot/modules/custom/va_gov_flags/va_gov_flags.module
./docroot/modules/custom/va_gov_flags/src/Controller/VaGovFlagsController.php
./docroot/modules/custom/va_gov_flags/src/FeatureFlagDataBuilder.php
./docroot/modules/custom/va_gov_flags/src/Export/ExportFeature.php
./docroot/modules/custom/va_gov_flags/src/Export/ExportFeatureInterface.php
./docroot/modules/custom/va_gov_flags/src/FeatureFlagDataBuilderInterface.php
./docroot/modules/custom/va_gov_flags/src/EventSubscriber/FeatureToggleSubscriber.php
./docroot/modules/custom/va_gov_migrate/va_gov_migrate.module
./docroot/modules/custom/va_gov_migrate/src/Form/MigrateConfigForm.php
./docroot/modules/custom/va_gov_migrate/src/Paragraph/NumberCallout.php
./docroot/modules/custom/va_gov_migrate/src/Paragraph/QaSection.php
./docroot/modules/custom/va_gov_migrate/src/Paragraph/AdditionalInformation.php
./docroot/modules/custom/va_gov_migrate/src/Paragraph/QAAccordion.php
./docroot/modules/custom/va_gov_migrate/src/Paragraph/Alert.php
./docroot/modules/custom/va_gov_migrate/src/Paragraph/QAUnstructured.php
./docroot/modules/custom/va_gov_migrate/src/Paragraph/CollapsiblePanelItem.php
./docroot/modules/custom/va_gov_migrate/src/Paragraph/LinksList.php
./docroot/modules/custom/va_gov_migrate/src/Paragraph/QASchema.php
./docroot/modules/custom/va_gov_migrate/src/Paragraph/CollapsiblePanel.php
./docroot/modules/custom/va_gov_migrate/src/Paragraph/ReactWidget.php
./docroot/modules/custom/va_gov_migrate/src/Paragraph/ExpandableText.php
./docroot/modules/custom/va_gov_migrate/src/Paragraph/Table.php
./docroot/modules/custom/va_gov_migrate/src/Paragraph/Base/QABase.php
./docroot/modules/custom/va_gov_migrate/src/Paragraph/LinksListItem.php
./docroot/modules/custom/va_gov_migrate/src/Paragraph/Process.php
./docroot/modules/custom/va_gov_migrate/src/AnomalyMessage.php
./docroot/modules/custom/va_gov_migrate/src/Plugin/migrate_plus/data_parser/Csv.php
./docroot/modules/custom/va_gov_migrate/src/Plugin/migrate/source/SupportService.php
./docroot/modules/custom/va_gov_migrate/src/Plugin/migrate/source/AlertBlockSource.php
./docroot/modules/custom/va_gov_migrate/src/Plugin/migrate/source/VaMenuBase.php
./docroot/modules/custom/va_gov_migrate/src/Plugin/migrate/source/MetalsmithSource.php
./docroot/modules/custom/va_gov_migrate/src/Plugin/migrate/source/VaBenefitsMenu.php
./docroot/modules/custom/va_gov_migrate/src/Plugin/migrate/source/VaMainMenu.php
./docroot/modules/custom/va_gov_migrate/src/Plugin/migrate/process/VATablefieldHours.php
./docroot/modules/custom/va_gov_migrate/src/Plugin/migrate/process/VAFieldOfficeHours.php
./docroot/modules/custom/va_gov_migrate/src/ParagraphType.php
./docroot/modules/custom/va_gov_migrate/src/ParagraphMigrator.php
./docroot/modules/custom/va_gov_migrate/src/Commands/Commands.php
./docroot/modules/custom/va_gov_migrate/src/EventSubscriber/PrepareRow.php
./docroot/modules/custom/va_gov_migrate/src/EventSubscriber/PostRowSave.php
./docroot/modules/custom/va_gov_migrate/src/EventSubscriber/MessageLogger.php
./docroot/modules/custom/va_gov_migrate/src/Obtainer/ObtainIframeVideo.php
./docroot/modules/custom/va_gov_migrate/src/Obtainer/ObtainTitleNoCaseChange.php
./docroot/modules/custom/va_gov_migrate/src/Obtainer/ObtainAndTestPlainTextWithNewLines.php
./docroot/modules/custom/va_gov_migrate/src/Obtainer/ObtainAndTestBody.php
./docroot/modules/custom/va_gov_migrate/src/Obtainer/ObtainAlertBlockTitles.php
./docroot/modules/custom/va_gov_migrate/src/Obtainer/ObtainAndTestFeature.php
./docroot/modules/custom/va_gov_bulk/src/Plugin/Action/PublishLatestRevisionAction.php
./docroot/modules/custom/va_gov_bulk/src/Plugin/Action/ArchiveNodeAction.php
./docroot/modules/custom/va_gov_bulk/src/Service/ModerationActions.php
./docroot/modules/custom/va_gov_bulk/src/Service/ModerationActionsInterface.php
./docroot/modules/custom/va_gov_help_center/va_gov_help_center.module
./docroot/modules/custom/va_gov_help_center/src/Plugin/Block/WhatsNew.php
./docroot/modules/custom/va_gov_events/va_gov_events.module
./docroot/modules/custom/va_gov_events/src/Form/SmartDateRemoveInstanceForm.php
./docroot/modules/custom/va_gov_events/src/Form/SmartDateOverrideDeleteAjaxForm.php
./docroot/modules/custom/va_gov_events/src/Form/SmartDateOverrideForm.php
./docroot/modules/custom/va_gov_events/src/Controller/Instances.php
./docroot/modules/custom/va_gov_events/src/Routing/RouteSubscriber.php
./docroot/modules/custom/va_gov_events/src/EventSubscriber/EntityEventSubscriber.php
./docroot/modules/custom/va_gov_build_trigger/src/RouteProvider.php
./docroot/modules/custom/va_gov_build_trigger/src/Form/BrdBuildTriggerForm.php
./docroot/modules/custom/va_gov_build_trigger/src/Form/BuildTriggerForm.php
./docroot/modules/custom/va_gov_build_trigger/src/Form/LocalBuildTriggerForm.php
./docroot/modules/custom/va_gov_build_trigger/src/Form/TugboatBuildTriggerForm.php
./docroot/modules/custom/va_gov_build_trigger/src/Form/PreviewForm.php
./docroot/modules/custom/va_gov_build_trigger/src/Traits/RunsDuringBusinessHours.php
./docroot/modules/custom/va_gov_build_trigger/src/Plugin/AdvancedQueue/JobType/ReleaseDispatch.php
./docroot/modules/custom/va_gov_build_trigger/src/Plugin/AdvancedQueue/JobType/ReleaseRequest.php
./docroot/modules/custom/va_gov_build_trigger/src/Plugin/MetricsCollector/TimeSinceLastContentReleaseError.php
./docroot/modules/custom/va_gov_build_trigger/src/Plugin/MetricsCollector/TimeSinceLastContentRelease.php
./docroot/modules/custom/va_gov_build_trigger/src/Plugin/MetricsCollector/ContentReleaseDurationRollingAverage.php
./docroot/modules/custom/va_gov_build_trigger/src/Plugin/MetricsCollector/ContentReleaseDuration.php
./docroot/modules/custom/va_gov_build_trigger/src/Plugin/MetricsCollector/ContentReleaseIntervalRollingAverage.php
./docroot/modules/custom/va_gov_build_trigger/src/Plugin/MetricsCollector/BaseContentReleaseMetricsCollector.php
./docroot/modules/custom/va_gov_build_trigger/src/Plugin/MetricsCollector/ContentReleaseInterval.php
./docroot/modules/custom/va_gov_build_trigger/src/Plugin/MetricsCollector/ContentReleaseAttemptsSinceLastSuccess.php
./docroot/modules/custom/va_gov_build_trigger/src/Plugin/VAGov/Environment/Tugboat.php
./docroot/modules/custom/va_gov_build_trigger/src/Plugin/VAGov/Environment/Local.php
./docroot/modules/custom/va_gov_build_trigger/src/Plugin/VAGov/Environment/BRD.php
./docroot/modules/custom/va_gov_build_trigger/src/Plugin/Block/ContentReleaseStatusBlock.php
./docroot/modules/custom/va_gov_build_trigger/src/Controller/ContentReleaseRequestController.php
./docroot/modules/custom/va_gov_build_trigger/src/Controller/FrontEndBranchAutocompleteController.php
./docroot/modules/custom/va_gov_build_trigger/src/Controller/ContentReleaseStatusBlockController.php
./docroot/modules/custom/va_gov_build_trigger/src/Controller/PlaceholderController.php
./docroot/modules/custom/va_gov_build_trigger/src/Controller/ContentReleaseNotificationController.php
./docroot/modules/custom/va_gov_build_trigger/src/SiteStatus/SiteStatus.php
./docroot/modules/custom/va_gov_build_trigger/src/SiteStatus/SiteStatusInterface.php
./docroot/modules/custom/va_gov_build_trigger/src/Environment/EnvironmentInterface.php
./docroot/modules/custom/va_gov_build_trigger/src/Environment/EnvironmentManager.php
./docroot/modules/custom/va_gov_build_trigger/src/Environment/EnvironmentPluginBase.php
./docroot/modules/custom/va_gov_build_trigger/src/Environment/Annotation/Environment.php
./docroot/modules/custom/va_gov_build_trigger/src/Environment/EnvironmentDiscovery.php
./docroot/modules/custom/va_gov_build_trigger/src/Commands/ContentReleaseCommands.php
./docroot/modules/custom/va_gov_build_trigger/src/Commands/SiteStatusCommands.php
./docroot/modules/custom/va_gov_build_trigger/src/Service/BuildRequester.php
./docroot/modules/custom/va_gov_build_trigger/src/Service/ReleaseStateManagerInterface.php
./docroot/modules/custom/va_gov_build_trigger/src/Service/BuildRequesterInterface.php
./docroot/modules/custom/va_gov_build_trigger/src/Service/BuildSchedulerInterface.php
./docroot/modules/custom/va_gov_build_trigger/src/Service/BuildScheduler.php
./docroot/modules/custom/va_gov_build_trigger/src/Service/ReleaseStateManager.php
./docroot/modules/custom/va_gov_build_trigger/src/EventSubscriber/EntityEventSubscriber.php
./docroot/modules/custom/va_gov_build_trigger/src/EventSubscriber/ContinuousReleaseSubscriber.php
./docroot/modules/custom/va_gov_build_trigger/src/EventSubscriber/ContentReleaseIntervalSubscriber.php
./docroot/modules/custom/va_gov_build_trigger/src/EventSubscriber/ContentReleaseErrorSubscriber.php
./docroot/modules/custom/va_gov_build_trigger/src/EventSubscriber/ContentReleaseMetricsRecalculationSubscriber.php
./docroot/modules/custom/va_gov_build_trigger/src/EventSubscriber/RouteSubscriber.php
./docroot/modules/custom/va_gov_build_trigger/src/Event/ReleaseStateTransitionEvent.php
./docroot/modules/custom/va_gov_workflow_assignments/va_gov_workflow_assignments.module
./docroot/modules/custom/va_gov_workflow_assignments/src/Plugin/Block/EntityMetaDisplay.php
./docroot/modules/custom/va_gov_workflow_assignments/src/Service/EditorialWorkflowContentRepository.php
./docroot/modules/custom/va_gov_workflow_assignments/src/Service/EditorialWorkflowContentRepositoryInterface.php
./docroot/modules/custom/va_gov_workflow_assignments/src/EventSubscriber/EntityEventSubscriber.php
./docroot/modules/custom/va_gov_menu_access/va_gov_menu_access.module
./docroot/modules/custom/va_gov_menu_access/src/Form/VaGovMenuAccessConfigForm.php
./docroot/modules/custom/va_gov_menu_access/src/Access/RouteAccessChecks.php
./docroot/modules/custom/va_gov_menu_access/src/Routing/AdminRouteSubscriber.php
./docroot/modules/custom/va_gov_menu_access/src/Service/MenuReductionService.php
./docroot/modules/custom/va_gov_db/va_gov_db.post_update.php
./docroot/modules/custom/va_gov_db/va_gov_db.module
./docroot/modules/custom/va_gov_backend/va_gov_backend.module
./docroot/modules/custom/va_gov_backend/modules/va_gov_backend_http_client/src/VaGovBackendHttpClient.php
./docroot/modules/custom/va_gov_backend/src/Form/ExclusionTypesAdminForm.php
./docroot/modules/custom/va_gov_backend/src/Logger/Processor/DatadogApmProcessor.php
./docroot/modules/custom/va_gov_backend/src/Test/DeployPluginTest.php
./docroot/modules/custom/va_gov_backend/src/Test/DeployServiceMock.php
./docroot/modules/custom/va_gov_backend/src/Deploy/Plugin/MaintenanceMode.php
./docroot/modules/custom/va_gov_backend/src/Deploy/Plugin/FeatureFlag.php
./docroot/modules/custom/va_gov_backend/src/Deploy/Plugin/HealthCheck.php
./docroot/modules/custom/va_gov_backend/src/Deploy/Plugin/DeployPluginInterface.php
./docroot/modules/custom/va_gov_backend/src/Deploy/DeployService.php
./docroot/modules/custom/va_gov_backend/src/Deploy/LoadMaintenanceFileTrait.php
./docroot/modules/custom/va_gov_backend/src/Deploy/SuccessHTTPException.php
./docroot/modules/custom/va_gov_backend/src/Deploy/FileOperationsTrait.php
./docroot/modules/custom/va_gov_backend/src/Plugin/field_group/FieldGroupFormatter/Tooltip.php
./docroot/modules/custom/va_gov_backend/src/Plugin/field_group/FieldGroupFormatter/DetailsWithImage.php
./docroot/modules/custom/va_gov_backend/src/Plugin/MetricsCollector/ActiveUserCount.php
./docroot/modules/custom/va_gov_backend/src/Plugin/Field/FieldWidget/TooltipTextFieldFormatter.php
./docroot/modules/custom/va_gov_backend/src/Plugin/Filter/NodeLinkEnforcementFilter.php
./docroot/modules/custom/va_gov_backend/src/Plugin/Filter/EmailLinkRepairFilter.php
./docroot/modules/custom/va_gov_backend/src/Plugin/Validation/Constraint/RequiredParagraph.php
./docroot/modules/custom/va_gov_backend/src/Plugin/Validation/Constraint/PdfCheckValidator.php
./docroot/modules/custom/va_gov_backend/src/Plugin/Validation/Constraint/PreventAbsoluteCmsLinksValidator.php
./docroot/modules/custom/va_gov_backend/src/Plugin/Validation/Constraint/Traits/ValidatorContextAccessTrait.php
./docroot/modules/custom/va_gov_backend/src/Plugin/Validation/Constraint/PreventProtocolRelativeLinks.php
./docroot/modules/custom/va_gov_backend/src/Plugin/Validation/Constraint/PreventLocalFileLinksValidator.php
./docroot/modules/custom/va_gov_backend/src/Plugin/Validation/Constraint/BenefitsSelectionLimit.php
./docroot/modules/custom/va_gov_backend/src/Plugin/Validation/Constraint/UniqueTitle.php
./docroot/modules/custom/va_gov_backend/src/Plugin/Validation/Constraint/PreventLocalFileLinks.php
./docroot/modules/custom/va_gov_backend/src/Plugin/Validation/Constraint/ValidPhoneNumber.php
./docroot/modules/custom/va_gov_backend/src/Plugin/Validation/Constraint/BenefitsSelectionLimitValidator.php
./docroot/modules/custom/va_gov_backend/src/Plugin/Validation/Constraint/ValidPhoneNumberValidator.php
./docroot/modules/custom/va_gov_backend/src/Plugin/Validation/Constraint/PdfCheck.php
./docroot/modules/custom/va_gov_backend/src/Plugin/Validation/Constraint/PreventProtocolRelativeLinksValidator.php
./docroot/modules/custom/va_gov_backend/src/Plugin/Validation/Constraint/UniqueTitleValidator.php
./docroot/modules/custom/va_gov_backend/src/Plugin/Validation/Constraint/RequiredParagraphValidator.php
./docroot/modules/custom/va_gov_backend/src/Plugin/Validation/Constraint/PreventAbsoluteCmsLinks.php
./docroot/modules/custom/va_gov_backend/src/Plugin/Block/SidebarMenusBlock.php
./docroot/modules/custom/va_gov_backend/src/Plugin/media/Source/ExternalFile.php
./docroot/modules/custom/va_gov_backend/src/Controller/VaGovBackendController.php
./docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php
./docroot/modules/custom/va_gov_backend/src/Commands/MetricsCommands.php
./docroot/modules/custom/va_gov_backend/src/Routing/RouteSubscriber.php
./docroot/modules/custom/va_gov_backend/src/Service/ExclusionTypesInterface.php
./docroot/modules/custom/va_gov_backend/src/Service/DatadogContextProvider.php
./docroot/modules/custom/va_gov_backend/src/Service/VaGovUrl.php
./docroot/modules/custom/va_gov_backend/src/Service/ExclusionTypes.php
./docroot/modules/custom/va_gov_backend/src/Service/Metrics.php
./docroot/modules/custom/va_gov_backend/src/Service/DatadogContextProviderInterface.php
./docroot/modules/custom/va_gov_backend/src/Service/Datadog.php
./docroot/modules/custom/va_gov_backend/src/Service/VaGovUrlInterface.php
./docroot/modules/custom/va_gov_backend/src/EventSubscriber/EntityEventSubscriber.php
./docroot/modules/custom/va_gov_notifications/src/Plugin/Notifier/Slack.php
./docroot/modules/custom/va_gov_notifications/src/Service/FlaggingInterface.php
./docroot/modules/custom/va_gov_notifications/src/Service/Flagging.php
./docroot/modules/custom/va_gov_notifications/src/Service/NotificationsManager.php
./docroot/modules/custom/va_gov_notifications/src/Event/Entity/EditedFlag.php
./docroot/modules/custom/va_gov_media/va_gov_media.module
./docroot/modules/custom/va_gov_dashboards/va_gov_dashboards.module
./docroot/modules/custom/va_gov_dashboards/src/Plugin/Derivative/VcDashboardsBlock.php
./docroot/modules/custom/va_gov_dashboards/src/Plugin/Block/VcDashboardsBlock.php
./docroot/modules/custom/va_gov_dashboards/src/Service/VetCenterDashboard.php
./docroot/modules/custom/va_gov_login/va_gov_login.module
./docroot/modules/custom/va_gov_vamc/va_gov_vamc.module
./docroot/modules/custom/va_gov_vamc/src/Controller/EntityAutocompleteController.php
./docroot/modules/custom/va_gov_vamc/src/EntityAutocompleteMatcher.php
./docroot/modules/custom/va_gov_vamc/src/Routing/AutocompleteRouteSubscriber.php
./docroot/modules/custom/va_gov_vamc/src/Service/ContentHardeningDeduper.php
./docroot/modules/custom/va_gov_vamc/src/EventSubscriber/EntityEventSubscriber.php
./docroot/modules/custom/va_gov_facilities/va_gov_facilities.module
./docroot/modules/custom/va_gov_facilities/src/EventSubscriber/FacilitiesSubscriber.php
./docroot/modules/custom/va_gov_vet_center/va_gov_vet_center.module
./docroot/modules/custom/va_gov_vet_center/src/Plugin/QueueWorker/CronRequiredService.php
./docroot/modules/custom/va_gov_vet_center/src/Plugin/QueueWorker/RequiredServiceBase.php
./docroot/modules/custom/va_gov_vet_center/src/Service/RequiredServices.php
./docroot/modules/custom/va_gov_vet_center/src/EventSubscriber/EntityEventSubscriber.php
./docroot/modules/custom/va_gov_post_api/va_gov_post_api.module
./docroot/modules/custom/va_gov_post_api/src/Form/VaGovFacilityForceQueueForm.php
./docroot/modules/custom/va_gov_post_api/src/Form/VaGovPostApiSettingsForm.php
./docroot/modules/custom/va_gov_post_api/src/Service/PostFacilityService.php
./docroot/modules/custom/va_gov_post_api/src/Service/PostFacilityStatus.php
./docroot/modules/custom/va_gov_post_api/src/Service/PostFacilityBase.php
./docroot/modules/custom/va_gov_post_api/src/EventSubscriber/QueueProcessedEventSubscriber.php
./docroot/modules/custom/va_gov_post_api/src/EventSubscriber/QueueItemProcessedEventSubscriber.php
./docroot/modules/custom/va_gov_links/src/EventSubscriber/ContentReleaseBrokenLinksIngestion.php
./docroot/modules/custom/va_gov_clone/src/CloneManagerInterface.php
./docroot/modules/custom/va_gov_clone/src/CloneManager.php
./docroot/modules/custom/va_gov_clone/src/Plugin/VAGov/CloneHandler/Section.php
./docroot/modules/custom/va_gov_clone/src/Plugin/VAGov/CloneHandler/CLP.php
./docroot/modules/custom/va_gov_clone/src/CloneEntityFinder/CloneEntityFinderDiscovery.php
./docroot/modules/custom/va_gov_clone/src/CloneEntityFinder/Annotation/CloneEntityFinder.php
./docroot/modules/custom/va_gov_clone/src/CloneEntityFinder/CloneEntityFinderBase.php
./docroot/modules/custom/va_gov_clone/src/CloneEntityFinder/CloneEntityFinderInterface.php
./docroot/modules/custom/va_gov_clone/src/Commands/VaGovCloneCommands.php
```

Not _all_ of these can efficiently/effectively/valuably tested, but most should.

**Comment by ndouglas** [comment]
Opened followups:
- #10964 to improve coverage of what currently constitutes the `.module` files.
- #10966 to improve coverage of classes.
